src/src/components/navigation/NavigationMenu.vue

```diff
@@ -0,0 +1,50 @@
+<template>
+  <ul :class="customClass">
+    <NavigationMenuItem
+      v-for="item in items"
+      :id="item.id"
+      :key="item.id"
+      :name="item.name"
+    />
+  </ul>
+</template>
+
+<script>
+import { defineComponent, computed } from "vue";
+import NavigationMenuItem from "./NavigationMenuItem.vue";
+
+export default defineComponent({
+  name: "Navigation Menu",
```

Comment:
Codacy has a fix for the issue: [Property name "Navigation Menu" is not PascalCase.](https://app.codacy.com/gh/Supershift/Sushi.Mediakiwi.UI/pullRequest?prid=8211123)
```suggestion
  name: "Navigation Menu",
```